src/cmdlib.sh

```diff
@@ -255,6 +255,33 @@ EOF
     return "$(cat "${workdir}"/tmp/rc)"
 }
 
+openshift_git_hack() {
+    # When OPENSHIFT_GIT_HACK is defined as a build environment variable,
```

Comment:
where are GIT_URL and GIT_REF defined?

Reply:
They'll have to be defined as part of the `BuildConfig` in OpenShift

Reply:
`GIT_URL`, `GIT_REF` and `OPENSHIFT_GIT_HACK` are defined externally in the Openshift `buildconfig` for the Image Stream. Running `podman -e GIT_URL=....` would work too. I wanted to make sure this "hack" would only ever be hit when it was intended.

Reply:
can we update the comment to mention they are defined in the build environment?

Reply:
done